### PR TITLE
Lighthouse jan 2017 update

### PR DESCRIPTION
**Pull request by ebidel**
R: @kaycebasques 

GDoc: https://goo.gl/vxhXTc

Also added `devsite-crx-install` to the LH page so we're prepped for the inline install flow that's landing soon in devsite!

### CODE REVIEW

**Comment by petele** [comment]
@ebidel - if you create this PR from google/webfundamentals instead of your own fork, it'll create a live staged version of the PR for you...

**Comment by ebidel** [comment]
Oh nice. I'll do that next time.

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+![New documentation](../../images/2017/02/lighthouse/image_11.png)
+
+That's it for now!
+
+For all the details on the latest in Lighthouse, see the
+[full release notes](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0)
+over on Github. As always, [hit us up](https://github.com/GoogleChrome/lighthouse/graphs/contributors)
+to [report bugs](https://github.com/GoogleChrome/lighthouse/issues), file feature requests,
+or brainstorm [ideas](https://github.com/GoogleChrome/lighthouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+bug%22) on what you'd like to see next.
+
```
Comment (addyosmani) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Totally up to you: I might suggest giving a call-out to folks on the team that worked on the release for good feels (e.g 'With thanks to Brendan, Paul, Patrick etc... who helped make this release awesome').

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+over the sharing of the report and you can delete it at any time. You can revoke
+the Viewer's permission to create gists under your [GitHub settings](https://github.com/settings/applications).
+
+## Performance Experiment
+
+The first version of the [Performance Experiment](https://github.com/GoogleChrome/lighthouse/issues/1143)
+project has landed in 1.5.0. This lets you experiment with your page load performance,
+interactively testing the effects of blocking or delaying assets in your critical
+path **during development**.
+
```
Comment (addyosmani) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Is there value in linking up to https://docs.google.com/document/d/1FYt5Es_Kf5IyC_bkTHj2G_a_sTvRvIq5iZCEN8VZY5o/edit#heading=h.cetla8h0y4o for folks that don't know what the feature does behind the scenes?

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+One difference between Lighthouse runs on the command line versus in the
+extension is the fact that the CLI version can disable many more features of
+Chrome that may interfere with measuring performance, not the least of which
+are other extensions.
+
+In an attempt to make Lighthouse more consistent everywhere it's used, the
+extension's settings now has an option to disabled other Chrome extensions
+before running Lighthouse. If we have problems auditing your page, try enabling
+this option to see if it fixes the issue.
```
Comment (addyosmani) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: 🎉

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+     class="lighthouse-logo attempt-right" alt="Lighthouse Logo">
+
+<style>
+.lighthouse-logo {
+  height: 150px;
+  width: auto;
+}
+</style>
+
+Today, we're happy to announce the [1.5 release](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0) of Lighthouse, a huge release, with over **128 PRs**. Lighthouse 1.5 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (`npm i -g lighthouse`) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
```
Comment (petele) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Please break lines at 80 chars

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+![New documentation](../../images/2017/02/lighthouse/image_11.png)
+
+That's it for now!
+
+For all the details on the latest in Lighthouse, see the
+[full release notes](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0)
+over on Github. As always, [hit us up](https://github.com/GoogleChrome/lighthouse/graphs/contributors)
+to [report bugs](https://github.com/GoogleChrome/lighthouse/issues), file feature requests,
+or brainstorm [ideas](https://github.com/GoogleChrome/lighthouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+bug%22) on what you'd like to see next.
+
```
Comment (petele) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Do you want to include the comment widget?

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+</style>
+
+Today, we're happy to announce the [1.5 release](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0) of Lighthouse, a huge release, with over **128 PRs**. Lighthouse 1.5 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (`npm i -g lighthouse`) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
+
+## New Audits
+
+The **CSS Usage Audit** reports the number of unused styles rules in your page
+and the cost/time savings of removing them:
+
+## ![CSS Usage Audit](../../images/2017/02/lighthouse/image_1.png)
```
Comment (petele) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Small preference for absolute URLs instead of relative. If we ever change file paths again, fixing relative is a pita.

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+  height: 150px;
+  width: auto;
+}
+</style>
+
+Today, we're happy to announce the [1.5 release](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0) of Lighthouse, a huge release, with over **128 PRs**. Lighthouse 1.5 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (`npm i -g lighthouse`) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
+
+## New Audits
+
+The **CSS Usage Audit** reports the number of unused styles rules in your page
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/unused styles rules/unused style rules/

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+The **Deprecations and Interventions Audit** lists console warnings from Chrome
+if your page is using deprecated APIs or features that have
+[interventions](https://www.chromestatus.com/features#intervention):
+
+![Deprecations and Interventions Audit](../../images/2017/02/lighthouse/image_4.png)
+
+## Report changes
+
+As you've seen, we've focused on making the report less visually cluttered by
+adding tabular data, hiding extraneous help text, and added new features to make
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/added/adding/

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+that were used to create the report; a [long time feature request](https://github.com/GoogleChrome/lighthouse/issues/568):
+
+![Emulation settings](../../images/2017/02/lighthouse/image_5.gif)
+
+### More useful trace data
+
+Lighthouse metrics like "First meaningful paint", "Time to Interactive", etc are
+mocked out as User Timing measures and injected back into the trace data saved with the `--save-assets` flag.
+
+If you use the  `--save-assets` flag when running Lighthouse, drop the trace into
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: This doesn't read like a complete sentence to me... maybe reword to "If you use the --save-assets flag, you can now drop the trace into DevTools or open it in the DevTools Timeline Viewer"

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+Extension users may notice Chrome warning about a change in permissions. You may need to re-enable the Lighthouse extension after Chrome auto-updates it.
+{: .note }
+
+One difference between Lighthouse runs on the command line versus in the
+extension is the fact that the CLI version can disable many more features of
+Chrome that may interfere with measuring performance, not the least of which
+are other extensions.
+
+In an attempt to make Lighthouse more consistent everywhere it's used, the
+extension's settings now has an option to disabled other Chrome extensions
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/disabled/disable/

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+before running Lighthouse. If we have problems auditing your page, try enabling
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/we/Lighthouse/

"We" implies that you and Brendan are doing the auditing, not LH

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+this option to see if it fixes the issue.
+
+![Disable other extensions setting](../../images/2017/02/lighthouse/image_7.png)
+
+## Lighthouse Viewer
+
+In HTML reports, you'll notice a new button with options for exporting the
+report in different formats. One of those options is "Open in Viewer". Clicking
+this button will send the report to the online [Viewer]([https://googlechrome.github.io/lighthouse/viewer/](https://googlechrome.github.io/lighthouse/viewer/)),
+where you can further share the report with users using Github.
```
Comment (kaycebasques) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/users using GitHub/GitHub users/

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 2 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Lighthouse 1.5.
+
+{# wf_updated_on: 2017-02-10 #}
+{# wf_published_on: 2017-02-10 #}
+{# wf_tags: lighthouse,tools,performance,progressive-web-apps #}
+{# wf_featured_image: /web/progressive-web-apps/images/pwa-lighthouse.png #}
+{# wf_featured_snippet: What's new in Lighthouse 1.5. New audits, extension updates, Performance Experiment, online Viewer features, and UI tweaks. #}
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Performance Experiment isn't a proper noun. Don't capitalize.

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+# Lighthouse January 2017 update {: .page-title }
+
+{% include "web/_shared/contributors/brendankenny.html" %}
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+[Lighthouse](/web/tools/lighthouse/) is an
+[open-source](https://github.com/GoogleChrome/lighthouse), automated tool for
+improving the quality of your web apps. You can install it as a
+[Chrome Extension][crx] or run it as a Node command line tool. When you
+give Lighthouse a URL, it runs a barrage of tests against the page and then
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Optional: consider saying 'battery' instead of 'barrage'.

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 6 above)
Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Absolute starting from `/web/updates/` etc., not from https://developers.google.com.

Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 5 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 7 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 8 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 9 above)
Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: It is a complete sentence. Basically, it's ths: *If you use the flag, drop the trace.*

Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 10 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: 'has' to 'have'. It has to agree with 'extension' which is singular.

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 11 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: +1

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 4 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done.

I'll express my frustration again with an 80 char wrap for text documents. Especially without a lint CI check. It's tedious to author a document when most of your time is spent wrapping and rewrapping lines :\

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 1 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Saving shout outs for the release notes.

**Comment by ebidel** [comment]
PTAL

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 4 above)
Comment (petele) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: I understand. The main reason I push for this is that it reduces diff sizes, both here and in `g3`.  

A lint check has been on my list for some time, happy to chat about that if you're willing to take a peek.

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 14 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: Mostly a cut and paste from the previous post and https://developers.google.com/web/tools/lighthouse/

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+and the cost/time savings of removing them:
+
+## ![CSS Usage Audit](/web/updates/images/2017/02/lighthouse/image_1.png)
+
+The **Optimized Images Audit** reports images that are unoptimized and the
+cost/time savings of optimizing them:
+
+![Optimized Images AudiT](/web/updates/images/2017/02/lighthouse/image_2.png)
+
+The **Responsive Images Audit** reports images that are too big and their
```
Comment (brendankenny) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: s/their/the?

File: src/content/en/updates/2017/02/lighthouse-jan.md [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+if your page is using deprecated APIs or features that have
+[interventions](https://www.chromestatus.com/features#intervention):
+
+![Deprecations and Interventions Audit](/web/updates/images/2017/02/lighthouse/image_4.png)
+
+## Report changes
+
+As you've seen, we've focused on making the report less visually cluttered by
+adding tabular data, hiding extraneous help text, and adding new features to
+make navigating the data, easier.
```
Comment (brendankenny) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: no comma

File: src/data/_contributors.yaml [hunk 17]
```diff
@@ -136,8 +136,8 @@ ericbidelman:
     - author
   email: e.bidelman@google.com
   description:
-    en: 'Engineer @ Google working on Polymer, Chrome, and the web'
-  
+    en: 'Engineer @ Google working on Lighthouse, Web Components, Chrome, and the web'
```
Comment (brendankenny) [inline src/data/_contributors.yaml]: will you give me a cool description too? :)

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 15 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 16 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: reworded

File: src/data/_contributors.yaml (hunk 17 above)
Comment (ebidel) [inline src/data/_contributors.yaml]: what do you want it as?

Comment (ebidel) [inline src/data/_contributors.yaml]: "Lighthouse Bulb Installer"

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 2 above)
Comment (ebidel) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: done

File: src/data/_contributors.yaml (hunk 17 above)
Comment (brendankenny) [inline src/data/_contributors.yaml]: srl_lens

File: src/content/en/updates/2017/02/lighthouse-jan.md (hunk 4 above)
Comment (jpmedley) [inline src/content/en/updates/2017/02/lighthouse-jan.md]: I do it in Sublime. I set up a key command for the menu item. I still have to do a tiny bit of fussing around links since the need to be on one line. This works 99% of the time. Go to Sublime Text > Preferences > Key Bindings - User and add: 

```
[
	{ "keys": ["ctrl+w"], "command": "wrap_lines", "args": {"width": 80}}
]
```
